web/base_tree_viewer.js

```diff
@@ -43,14 +43,14 @@ class BaseTreeViewer {
   }
 
   /**
-   * @private
+   * @protected
```

Comment:
Here, and everywhere else throughout the patch, why do we need to make this change?

Reply:
cause typescript wise you can't redefine a private property in a child class (was leading to a broken .d.ts)